docs/tutorials/host_apt_news.rst

```diff
@@ -0,0 +1,282 @@
+
+.. code-block:: console
+
+   you@yourcomputer:~$ multipass shell tutorial
+
+When you see "Welcome to Ubuntu 24.04 LTS" and the shell prompt changes to the
+following, you know that you've entered the VM.
+
+.. code-block:: console
+
+   ubuntu@tutorial:~$
+
+Inside the VM, set up `LXD`_:
+
+.. code-block:: console
+
+   ubuntu@tutorial:~$ lxd init --minimal
+
+
+Launch three containers inside the VM with the following commands. This may
+take a while since it will need to download the appropriate Ubuntu images.
+
+.. code-block:: console
+
+   ubuntu@tutorial:~$ lxc launch ubuntu-daily:noble apt-news-server
+   ubuntu@tutorial:~$ lxc launch ubuntu-daily:jammy client-jammy
+   ubuntu@tutorial:~$ lxc launch ubuntu-daily:focal client-focal
+
+Set up the APT news server
+==========================
+
+Now, let's setup the ``apt-news-server`` container to serve APT news content.
+APT news content is formatted as a JSON file and served over HTTP(S). We can
+accomplish this by installing and configuring `nginx`_ to serve a properly
+formatted JSON file.
+
+First, enter the ``apt-news-server`` container:
+
+.. code-block:: console
+
+   ubuntu@tutorial:~$ lxc shell apt-news-server
+   root@apt-news-server:~#
+
+In ``apt-news-server``, install ``nginx`` via ``apt``. This will also start the HTTP server on port 80.
+
+.. code-block:: console
+
+   root@apt-news-server:~# apt install -y nginx
+
+With ``nginx`` installed and running, we can author an APT news JSON file. Open ``/var/www/html/aptnews.json``:
+
+.. code-block:: console
+
+   root@apt-news-server:~# nano /var/www/html/aptnews.json
+
+and add the following content:
+
+.. code-block:: json
+
+   {
+     "messages": [
+       {
+         "begin": "TODAY",
+         "selectors": {
+           "codenames": ["jammy"]
+         },
+         "lines": [
+           "Hello 22.04 users!",
+           "This is the APT news server."
+         ]
+       },
+       {
+         "begin": "TODAY",
+         "lines": [
+           "Hello everyone else!",
+           "This is the APT news server."
+         ]
+       }
+     ]
+   }
+
+In ``nano``, use CTRL-S and CTRL-X to save and exit, respectively.
+
+That apt news configuration will show one message to systems running Ubuntu
+22.04 (Codename "jammy") and will show another message to all other systems.
+
+The value of ``"begin"`` actually needs to be an ISO8601 formatted datetime
+string, and the message won't be shown before the ``begin`` date or more than 30
+days after the ``begin`` date. For the purposes of this tutorial you can
+quickly replace "TODAY" with today's date in ``aptnews.json`` with the
+following command.
+
+.. code-block:: console
+
+   root@apt-news-server:~# sed -i "s/TODAY/$(date --iso-8601=seconds)/" /var/www/html/aptnews.json
+
+You can double check that the command worked by verifying the ``begin`` field has an appropriate value.
+
+.. code-block:: console
+   :emphasize-lines: 5
```

Comment:
For some reason this makes the text italic - I'd remove it (we don't really need it)

Reply:
The `emphasize-lines` isn't what makes the text italic - I think it is the `console` type and that is unneeded for this code-block, I'll play with it.